src/styles/main.scss

```diff
@@ -1,3 +1,34 @@
 body {
   margin: 0;
 }
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+}
+
+.container__sub {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 100px;
+  color: #fff;
+  font-family: Arial, sans-serif;
+  height: 300px;
```

Comment:
It`s ok, but maybe use a variable for 300px?

Reply:
Good advice, thank you :). I added.